src/mocks/handlers/visiumHandlers.ts

```diff
@@ -53,9 +54,11 @@ const handlers = [
     }
 
     const labware = createLabware(barcode);
+    const samplePositionResults: SamplePositionFieldsFragment[] = [];
     return res(
       ctx.data({
         visiumPermData: {
+          samplePositionResults,
```

Comment:
This mocking is not correct as the core side doesn't support this

Reply:
The core code, is been updated accordingly, you can have a look here https://github.com/sanger/stan-core/pull/237